ts/Dashboards/EditMode/EditRenderer.ts

```diff
@@ -337,20 +518,28 @@ function renderText(
  */
 function renderIcon(
     parentElement: HTMLDOMElement,
-    icon: string,
-    callback?: Function
+    options: IconFormField
 ): HTMLDOMElement|undefined {
-    let iconElem;
+    const { icon, callback } = options;
 
-    if (parentElement) {
-        iconElem = createElement(
-            'div', {
-                onclick: callback
-            }, {},
-            parentElement
-        );
+    if (!parentElement) {
+        return;
+    }
 
-        (iconElem.style as any)['background-image'] = 'url(' + icon + ')';
+    const iconElem = createElement(
+        'div', {
+            onclick: callback
+        }, {},
+        parentElement
+    );
+
+    (iconElem.style as any)['background-image'] = 'url(' + icon + ')';
```

Comment:
Solve the `any`

Reply:
It already was here. I'll leave it as it is, and deal with it some other time.